### PR TITLE
Add course description for RSE Leadership Training.

### PR DESCRIPTION
**Pull request by NPDebs**
This PR resolves issue #791.

To the existing [consulting](https://openlifesci.org/consulting/organisational-leadership/) page, a section has been added, with information about the Research Software Engineer (RSE) Leadership Training Course.

[Deploy preview](https://deploy-preview-792--ols-bebatut.netlify.app/consulting/organisational-leadership/#research-software-engineer-rse-leadership-training-course) here...

**Note for reviewer(s)**
- I haven't linked to the training materials yet. Are we ready to go live with it.

Thanks for the review!

### CODE REVIEW

**Comment by bebatut** [comment]
I am not that should go there. It might be better as an event. What do you think?

**Comment by NPDebs** [comment]
> I am not that should go there. It might be better as an event. What do you think?

🤔 Good thinking.

I do feel like it fits into the current location. Could we do a compromise and also add it under `events`?

That doesn't make for DRY code, but it does give the program visibility everywhere that matters.

**Comment by bebatut** [comment]
My suggestion:

- Make a page in `events` for the specific event with dates 
- Add to the consulting page the description of the course but without details of the dates and add a list to list events running this course with link to the event page


Does that make sense?

**Comment by rowlandm** [comment]
@yochannah  and @NPDebs - I have an idea - please bear with me:

I'm thinking:
- This leadership course is possibly the first in the world to provide an intersectional framework that underpins a leadership course for those working as RSEs and those in the tech industry.
- It is also unique in that we meet with you first to understand what challenges you are facing and to to create an individual development plan just for you.
- We will then craft a practical training course just for you and a small group of other leaders to ensure all your needs are being met as an individual and as a group.
- Course will run weekly from 17th of October to 28th of November.

Topics covered include:
- Using a Diversity and Inclusion framework to unearth talent.
- Using Continuous Improvement skills to identify talent.
- Setup and maintain pipelines to attract talent.
- Project Management.
- People managing high-performance teams.
- How to deal with difficult change management situations and build relationships.

Pricing:
- £499 GBP for initial skills consult and personal development plan.
- £1999 GBP for the practical training course designed just for you and a small group of other leaders.

**Comment by NPDebs** [comment]
> Does that make sense?

It does, @bebatut. I'll implement and await your feedback. 👍

**Comment by NPDebs** [comment]
These are minor changes, @rowlandm. No worries at all. :)

I'd like to clarify one thing though.

Do you want to replace "**Couse content:**" with "**Topics covered include:**"?

**Comment by NPDebs** [comment]
> Make a page in events for the specific event with dates.

@bebatut I do have a concern about this approach. If we make a new `.md` file for each of these dates, we will end up having **unique** standalone events, when in reality, they are all part of a series.

It might be better to have a single `.md` file, which will ultimately contain all the details of the workshop.

Curious to hear what you think. 🙏

**Comment by NPDebs** [comment]
**First image: Events page**
Screenshot of events page

**Second image: RSE workshop page**
Screenshot of RSE workshop page


**Question:**
1. Who is the contact person for this workshop? Their email should go here: 👇 
        `contact_email: person@we-are-ols.org`

2. Will there be a deadine for registration? The registration link and deadline should be added here: 👇 
```
                 registration:
                         link: https://some-streaming-app/registration
                         deadline: YY-MM-DD
```

What questions do we have about the structure so far?

**Comment by rowlandm** [comment]
> These are minor changes, @rowlandm. No worries at all. :)
> 
> I'd like to clarify one thing though.
> 
> Do you want to replace "**Couse content:**" with "**Topics covered include:**"?

Yes please.

**Comment by rowlandm** [comment]
Feedback:

- Might want to move "LMIC discounts are available for all offers. If you are interested in any of the services, please contact us at team@we-are-ols.org." to the event page.
- Are you sure you want an event page instead of the registration page of the event on Humanitix when it is ready?

**Comment by NPDebs** [comment]
> Are you sure you want an event page instead of the registration page of the event on Humanitix when it is ready?

I was thinking we could have both. A main `events` page, from which we can link to the ticketing platform.
Does this approach sit well with you? @rowlandm

**Comment by rowlandm** [comment]
Yeah - I'm OK with that!

**Comment by rowlandm** [comment]
I'll be trying to setup the humanitix ticketing soon.

**Comment by rowlandm** [comment]
I've just changed the topics slightly:

- Using a Diversity and Inclusion framework to improve your team.
- Using Continuous Improvement skills to identify talent.
- Setup and maintain pipelines to attract talent.
- Project Management in complex scenarios.
- Managing high-performance teams.
- How to deal with difficult change management situations and build relationships.

Is it ok if they can go into this PR please?

**Comment by rowlandm** [comment]
I'm good with this now.

**Review by rowlandm — APPROVED** [review]
Thanks for all the work Debs - I'm good to approve this.

File: _consulting/organisational-leadership.md [hunk 1]
```diff
@@ -55,4 +55,23 @@ Country-specific nuances, e.g. setting up a [stichting](https://en.wikipedia.org
 
 **Costs: £3000**
 
-LMIC discounts are available for all offers. If you are interested in any of the services, please contact us at team@we-are-ols.org. 
+##  Research Software Engineer (RSE) Leadership Training Course 
+* **Course description:**
+  * This leadership course is possibly the first in the world to provide an **intersectional framework** that underpins a leadership course for those in the tech industry.
+  * It focuses on centering the individual and prioritising based on the "degree of difficulty" across recruitment, retention, and relationship management.
```
[the remark this answers is not in the document]
Comment (NPDebs) [reply _consulting/organisational-leadership.md:61]:
```suggestion
  * It focuses on centering the individual and prioritising based on the “degree of difficulty” across recruitment, retention, as well as project, people, and relationship management.
```

File: _events/2024-10-research-software-engineer-training-course.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  * Using a Diversity and Inclusion framework to improve your team.
+  * Using Continuous Improvement skills to identify talent.
+  * Setup and maintain pipelines to attract talent.
+  * Project Management in complex scenarios.
+  * Managing high-performance teams.
+  * How to deal with difficult change management situations and build relationships.
+
+* **Pricing:** 
+  * **£499 GBP** for initial skills consult and personal development plan.
+  * **£1999 GBP** ffor the practical training course designed just for you and a small group of other leaders.
```
Comment (NPDebs) [inline _events/2024-10-research-software-engineer-training-course.md:41]:
```suggestion
  * **£1999 GBP** for a series of workshops with a small number of other leaders, that is crafted to your needs based on the initial consultation.
```

File: _events/2024-10-research-software-engineer-training-course.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  deadline: 
+---
+
+* **Course description:**
+  * This leadership course is possibly the first in the world to provide an **intersectional framework** that underpins a leadership course for those working as RSEs and those in the tech industry.
+  * It is also unique in that we meet with you first to understand what challenges you are facing and to to create an individual development plan just for you.
+  * We will then craft a practical training course just for you and a small group of other leaders to ensure all your needs are being met as an individual and as a group.
+  * Course will run weekly from **17th of October** to **28th of November.**
+
+* **Topics covered include:**
```
Comment (NPDebs) [inline _events/2024-10-research-software-engineer-training-course.md:31]:
```suggestion
* **Workshop topics:**
```

File: _consulting/organisational-leadership.md [hunk 4]
```diff
@@ -55,4 +55,20 @@ Country-specific nuances, e.g. setting up a [stichting](https://en.wikipedia.org
...
+  * This course lays bare the core, practical concepts that you need to know to be an outstanding leader.
```
Comment (yochannah) [inline _consulting/organisational-leadership.md:62]:
```suggestion
  * This course lays bare the core, practical concepts that you need to know to be an outstanding leader for a team of 1+ staff in an organisation. 
```

To differentiate fro mteh open seeds program, which is about open community leaderdship.

Comment (yochannah) [reply _consulting/organisational-leadership.md:62]: let's hope "how to write without incredible typos" isn't in the course, I certainly am not capable :P

**Comment by yochannah** [comment]
Suggestions applied - I'm happy with everything here! :) I suspect we need to differentiate between this and open seeds a little, but that's not urgent.

**Comment by rowlandm** [comment]
I'm good with these changes. Thanks everyone!

**Review by yochannah — APPROVED** [review]

File: _events/2024-10-research-software-engineer-training-course.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+* Using a Diversity and Inclusion framework to improve your team.
+* Using Continuous Improvement skills to identify talent.
+* Setup and maintain pipelines to attract talent.
+* Project Management in complex scenarios.
+* Managing high-performance teams.
+* How to deal with difficult change management situations and build relationships.
+
+# Organization
+
+Course will run weekly from **17th of October** to **28th of November.**
```
Comment (bebatut) [inline _events/2024-10-research-software-engineer-training-course.md:45]: When will the course be during the weeks? How long will be the sessions?

**Comment by rowlandm** [comment]
Good question I think the days will either be possibly up to the group but
preferentially on Tuesdays or Fridays.

Duration would probably be an hour for the first part with the break of
maybe 5 to 10 minutes and then an hour for the second part so probably
about two and a half hours maximum.

**Review by yochannah — APPROVED** [review]

**Comment by yochannah** [comment]
If we incorporate the availability note from @rowlandm above in response to @bebatut, then I'm happy to merge this.

**Comment by yochannah** [comment]
Resolves #791

**Comment by NPDebs** [comment]
> If we incorporate the availability note from @rowlandm above in response to @bebatut, then I'm happy to merge this.

Yeah, but is it _Tuesdays, Fridays or based on participant availability_? 😅

**Comment by rowlandm** [comment]
Maybe: Tuesdays or Fridays, based on participant availability.

File: _events/2024-10-research-software-engineer-training-course.md [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
```
Comment (NPDebs) [inline _events/2024-10-research-software-engineer-training-course.md:46]:
```suggestion
Course will run weekly from **17th of October** to **28th of November, on Tuesdays or Fridays, based on participant availability.**
```

**Comment by NPDebs** [comment]
> Maybe: Tuesdays or Fridays, based on participant availability.

Thanks, Rowland! ✨

**Review by yochannah — COMMENTED** [review]
@rowlandm can we add the registration link and deadline to register by? 🙏

**Comment by rowlandm** [comment]
This is the [link to the Humanitix event](https://events.humanitix.com/rse-leadership-course).  The deadline for the event is set as 20th September 2024. You can see it on the Humanitix event, although it is not that clear.....

File: _events/2024-10-research-software-engineer-training-course.md [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+duration: "02:45:00"
+
+location:
+  name: Online
+
+contact_email: rowland@we-are-ols.org
+
+registration:
+  link: 
+  deadline:
```
Comment (bebatut) [inline _events/2024-10-research-software-engineer-training-course.md:23]:
```suggestion
  link: https://events.humanitix.com/rse-leadership-course
  deadline: 2024-09-24
```

File: _events/2024-10-research-software-engineer-training-course.md [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+* Using a Diversity and Inclusion framework to improve your team.
+* Using Continuous Improvement skills to identify talent.
+* Setup and maintain pipelines to attract talent.
+* Project Management in complex scenarios.
+* Managing high-performance teams.
+* How to deal with difficult change management situations and build relationships.
+
+# Organization
+
+Course will run weekly from **17th of October** to **28th of November, on Tuesdays or Fridays, based on participant availability.**
```
Comment (bebatut) [inline _events/2024-10-research-software-engineer-training-course.md:46]:
```suggestion
- **Dates**: from **17th of October** to **28th of November
- **Frequency**: weekly, on Tuesdays or Fridays, based on participant availability
- **Duration**: around two and a half hours
```

File: _events/2024-10-research-software-engineer-training-course.md [hunk 9]
```diff
@@ -0,0 +1,53 @@
...
+
+The course will cover
+* Using a Diversity and Inclusion framework to improve your team.
+* Using Continuous Improvement skills to identify talent.
+* Setup and maintain pipelines to attract talent.
+* Project Management in complex scenarios.
+* Managing high-performance teams.
+* How to deal with difficult change management situations and build relationships.
+
+# Organization
```
Comment (bebatut) [inline _events/2024-10-research-software-engineer-training-course.md:44]:
```suggestion
# Sessions
```

**Comment by NPDebs** [comment]
Thanks, everyone! Looks like we're good to merge now.

File: _events/2024-10-research-software-engineer-training-course.md [hunk 10]
```diff
@@ -0,0 +1,55 @@
...
+# Sessions
+
+- **Dates**: from **17th of October** to **28th of November
+- **Frequency**: weekly, on Tuesdays or Fridays, based on participant availability
+- **Duration**: around two and a half hours
+
+# Pricing
+
+* **£499 GBP** for initial skills consult and personal development plan.
+* **£1999 GBP** for a series of workshops with a small number of other leaders, that is crafted to your needs based on the initial consultation.
```
Comment (bebatut) [inline _events/2024-10-research-software-engineer-training-course.md:53]: Is this pricing for the sessions later this year?

Comment (rowlandm) [reply _events/2024-10-research-software-engineer-training-course.md:53]: Yes, the 1999 is for the workshops that are to happen on Tuesdays or Fridays.

**Pull request merged**